SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -357,13 +357,13 @@ HeroFusion = 1
 # Волшебное Зеркало работает и против войск ИИ игрока
 # Der Magische Spiegel des Spielers ist berücksichtigt, wenn die künstliche Intelligenz einen Zauberspruch wirkt
 # Το ξόρκι {Magic Mirror (Μαγικός Kαθρέφτης)} λαμβάνεται υπ' όψιν όταν η Τεχνητή Νοημοσύνη χρησιμοποιεί ένα ξόρκι
-AIMagicMirror = 1
+AIMagicMirror = 0
 
 # Enchanters will correctly consider all valid creatures on battlefield to cast their automatic spells
 # Автоматическое колдовство Чародеями будет производить проверку возможности каста заклинания на всю армию целевой стороны, а не только на первый стек.
 # Die Enchanters werden alle gültigen Kreaturen auf dem Schlachtfeld richtig berücksichtigen um ihre automatischen Zaubersprόche zu wirken
 # Οι Enchanters θα λαμβάνουν ορθά υπ' όψιν όλα τα έγκυρα πλάσματα στο πεδίο της μάχης για να εκτελέσουν τα αυτόματα ξόρκια τους
-EnchantersFix = 1
+EnchantersFix = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.